services/dns/record.go

```diff
@@ -14,8 +15,11 @@ import (
 var recordResourceName = "DNS Record"
 
 func (c *Client) CreateRecord(ctx context.Context, zoneId string, d *schema.ResourceData) (recordResponse dns.RecordResponse, responseInfo utils.ApiResponseInfo, err error) {
-	request := setRecordCreateRequest(d)
-	recordResponse, apiResponse, err := c.sdkClient.RecordsApi.ZonesRecordsPost(ctx, zoneId).RecordCreateRequest(*request).Execute()
+	// Must is a helper that wraps a call to a function returning (UUID, error) and panics if the error is non-nil.
+	uuidV4 := uuid.Must(uuid.NewV4())
+	recordUuid := uuid.NewV5(uuid.NewV5(uuid.NamespaceURL, "https://github.com/ionos-cloud/terraform-provider-ionoscloud"), uuidV4.String())
```

Comment:
I think we can extract this part as a constant? - uuid.NewV5(uuid.NamespaceURL, "https://github.com/ionos-cloud/terraform-provider-ionoscloud")